Jenkinsfile

```diff
@@ -29,12 +29,26 @@ pipeline {
 
         stage("Test conjur_host_identity role") {
           steps {
-            sh './ci/test.sh -d conjur_host_identity'
-            junit 'roles/conjur_host_identity/tests/junit/*'
+            // sh './ci/test.sh -d conjur_host_identity'
+            // junit 'roles/conjur_host_identity/tests/junit/*'
+            sh 'chmod +x ./tests/ansibletest.sh'
           }
         }
       }
     }
+    stage('Report Test Code Coverage'){
+          steps {
+            // dir('src/main/java'){
+            // ccCoverage('jacoco')
+            publishHTML (target : [allowMissing: false,
```

Comment:
This coverage report looks great. After looking at the coverage report from this branch, we should try to eliminate reporting for test files & empty files (like `test_conjur_variable.py` and `__init__.py`), and try to report coverage on only our source files.

Reply:
Done